src/containers/AssignmentTexterContact.jsx

```diff
@@ -112,7 +112,7 @@ export class AssignmentTexterContact extends React.Component {
 
   getMessageTextFromScript = script => {
     const { campaign, contact, texter } = this.props;
-
+    // console.log('ASSIGNMENT TEXTER CONTACT INITIATED', this.props)
```

Comment:
This console.log helped me see when applyScript was being instantiated by the AssignmentTexterContact